build.gradle

```diff
@@ -129,6 +130,12 @@ dependencies {
     compile group: 'com.microsoft.azure', name: 'applicationinsights-core', version: '1.0.9'
     compile group: 'com.microsoft.azure', name: 'applicationinsights-logging-log4j2', version: '1.0.9'
 
+    compile "org.glassfish.tyrus.bundles:tyrus-standalone-client:1.13.1"
+    compile "org.glassfish.tyrus.ext:tyrus-extension-deflate:1.13.1"
+
+    compile "com.google.code.gson:gson:2.8.0"
+    compile "org.bitbucket.cowwoc:diff-match-patch:1.1"
```

Comment:
Do we really need another diff library? Can't we use the diff library already used in the merge entries dialog? It is OK, if we do. Then, please explain it somehow. Preferrably using [MADR](https://github.com/adr/madr).

Reply:
Yes, this is diff library is the same which is used on the server side of sharelatex